bulk_insert.go

```diff
@@ -82,7 +82,8 @@ func insertObjSet(db *gorm.DB, objects []interface{}, excludeColumns ...string)
 		strings.Join(placeholders, ", "),
 	))
 
-	return db.Exec(mainScope.SQL, mainScope.SQLVars...).Error
+	db = db.Exec(mainScope.SQL, mainScope.SQLVars...)
+	return db.Error
```

Comment:
How is this any different? Instead of chaining you just assign the result from `Exec` before calling `Error`.

Could you give me an example of an error this change solves? To me this is just syntactical changes.

Reply:
[Here's](https://play.golang.org/p/2CLlGb0wdIX) a demonstration of the difference (and shows that the result is the same).

Reply:
Actually, you are right. I thought it was due to the error pointer wasn't returned. But looks like it's something else. Will close this. My apology..